test/utils/helpers.ts

```diff
@@ -72,6 +73,14 @@ export const filterByAmounts = (results: any, amounts: string[]) => {
 	})
   }
 
+export const filterByCounties = (results: any, counties: string[]) => {
+	if (!counties || !counties.length) {
+	  return results
+	}
+	
+	return results.filter((x: any) => counties.includes(x.county_council))
```

Comment:
as discussed earlier, this one also supposed to do the case insensitive comparison.

Reply:
This is only looking at the dummy data for testing purposes and doesn't need it.

Reply:
if you  take care it in the dummy data manually, that is ok as well.